sources/packages/web/src/views/aest/student/applicationDetails/ApplicationRestrictionsManagement.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <full-page-container layout-template="centered">
+    <template #header>
+      <header-navigator
+        title="Student Applications"
+        :routeLocation="{
+          name: AESTRoutesConst.STUDENT_APPLICATIONS,
+          params: { studentId },
+        }"
+        subTitle="Application Restriction Management"
+      />
+      <div v-if="applicationDetail.applicationNumber">
+        <span class="label-bold-normal">Application #: </span>
+        <span
+          >{{ emptyStringFiller(applicationDetail.applicationNumber) }} |
+        </span>
+        <span class="label-bold-normal">Institution: </span>
+        <span>{{ applicationDetail.applicationInstitutionName }} | </span>
+        <span class="label-bold-normal">Study Dates: </span>
+        <span
+          >{{ dateOnlyLongString(applicationDetail.applicationStartDate) }}
+          -
+          {{ dateOnlyLongString(applicationDetail.applicationEndDate) }}
+          |
+        </span>
+        <span class="label-bold-normal">Type: </span>
+        <span
+          >{{
+            getOfferingIntensity(applicationDetail.applicationOfferingIntensity)
+          }}
+        </span>
+      </div>
```

Comment:
I believe you can use the `<header-title-value>` component here.

Reply:
This component is replaced with `<application-header-title :application-id="applicationId" />`